frontend/src/pages/MomentCreationPage.js

```diff
@@ -41,23 +41,46 @@ class MomentCreation extends Component {
         });
         this.setState({redirect: '/'});
     }
+
+    handleChange = (event) => {
+        console.log(event.target.files[0].name);
+        const filename = event.target.files[0].name;
+        document.getElementById('filename').innerHTML = filename;
+    }
+
     render(){
         return(
             <div>
                 <NavBar user={this.state.user}/>
+                <div className = "createMoment-header">
+                <img
+                    alt="UploadMoment"
+                    src={`http://${process.env.REACT_APP_IP}/res/uploadMoment.png`}
```

Comment:
The image is pixelated... can't have that

Reply:
fixed it